src/Symfony/Component/VarDumper/Dumper/ServerDumper.php

```diff
@@ -39,7 +47,43 @@ public function __construct(string $host, DataDumperInterface $wrappedDumper = n
 
         $this->host = $host;
         $this->wrappedDumper = $wrappedDumper;
-        $this->contextProviders = $contextProviders;
+        $this->contextProviders = $contextProviders + self::getDefaultContextProviders();
+    }
+
+    /**
+     * @final
+     */
+    public static function getDefaultContextProviders(Request $request = null, string $projectDir = null): array
+    {
+        $contextProviders = array();
+
+        if ('cli' !== PHP_SAPI && ($request || class_exists(Request::class))) {
+            $requestStack = new RequestStack();
+            $requestStack->push($request ?? Request::createFromGlobals());
+            $contextProviders['request'] = new RequestContextProvider($requestStack);
+        }
+
+        $fileLinkFormatter = class_exists(FileLinkFormatter::class) ? new FileLinkFormatter(null, $requestStack ?? null, $projectDir) : null;
+        $contextProviders = array(
```

Comment:
There is a bug, you erase the $contextProviders that you define on line 63

you must do $contextProviders += array(

Reply:
Oops. Good catch. Refactoring issue. Thank you 👍